workers/generic-worker/multiuser_test.go

```diff
@@ -47,6 +48,31 @@ func TestWhoAmIAsCurrentUser(t *testing.T) {
 	_ = submitAndAssert(t, td, payload, "completed", "completed")
 }
 
+func TestTaskUserCredentialsEnvVarIsWrittenAsCurrentUser(t *testing.T) {
+	setup(t)
+	config.EnableRunTaskAsCurrentUser = true
```

Comment:
config is a global, do we need a defer block to restore the previous value? i don't remember. it is probably set in setup(t) but not all tests call setup(t) I think, so it feels dangerous to modify it but not restore it in a teardown.